install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -85,25 +94,93 @@ spec:
                 - probe
                 - --probe-path=/healthliveness
                 - --interval=10s
+            failureThreshold: 3
             initialDelaySeconds: 5
             periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 1
           readinessProbe:
             exec:
               command:
                 - /usr/local/bin/galley
                 - probe
                 - --probe-path=/healthready
                 - --interval=10s
+            failureThreshold: 3
             initialDelaySeconds: 5
             periodSeconds: 5
+            successThreshold: 1
+            timeoutSeconds: 1
           resources:
 {{- if .Values.resources }}
 {{ toYaml .Values.resources | indent 12 }}
 {{- else }}
 {{ toYaml .Values.global.defaultResources | indent 12 }}
+{{- end }}
+{{- if .Values.sidecar }}
+        - name: istio-proxy
+      {{- if contains "/" .Values.global.proxy.image }}
+          image: "{{ .Values.global.proxy.image }}"
+      {{- else }}
+          image: "{{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}"
+      {{- end }}
+          imagePullPolicy: {{ .Values.global.imagePullPolicy }}
+          ports:
+            - containerPort: 9902
+          {{- if not .Values.sidecar }}
```

Comment:
Will it be executed? given the while block is included in `{{- if .Values.sidecar }}`.